src/main/java/org/apache/commons/net/ftp/parser/MLSxEntryParser.java

```diff
@@ -64,10 +64,10 @@ public class MLSxEntryParser extends FTPFileEntryParserImpl {
 
     private static final HashMap<String, Integer> TYPE_TO_INT = new HashMap<>();
     static {
-        TYPE_TO_INT.put("file", Integer.valueOf(FTPFile.FILE_TYPE));
```

Comment:
Don't change this code: leave boxing/unboxing alone.

Reply:
Roger, will undo and push once more.